app/src/main/java/com/github/geohunt/app/ui/components/navigation/NavigationController.kt

```diff
@@ -104,7 +109,13 @@ fun NavigationController(
             )
         }
         composable(Route.ActiveHunts.route) {
-            Text("Active hunts")
+            val user = Authenticator.authInstance.get().user
+
+            if (user == null) {
+                Text("You are not logged in. Weird :(")
+            } else {
+                ActiveHunts(id = user.uid, database, controller)
```

Comment:
A similar comment to the `NavController` one: we shouldn't pass the `database` as an argument to `Composable`, and instead use a singleton instance. This would avoid accidentally creating multiple instances.

I think this problem will be solved once we have proper `ViewModel`s and `UiState`s...